src/lib/module_version/registration.ml

```diff
@@ -115,24 +196,27 @@ let%test_module "Test versioned modules" =
       module Registrar = Make (Module_decl)
       module Registered_V1 = Registrar.Register (V1)
       module Registered_V2 = Registrar.Register (V2)
-      module Registered_Latest = Registered_V2
     end
 
     open Stable
 
```

Comment:
can you add a test for serializing/deserializing nested ones? To make sure the "ignoring the version" logic is working?